### PR TITLE
RavenDB-16530 : OLAP testing

### PR DESCRIPTION
**Pull request by aviv86**

### CODE REVIEW

File: test/SlowTests/Server/Documents/ETL/Olap/LocalTests.cs [hunk 1]
```diff
@@ -932,7 +936,9 @@ public async Task AfterDatabaseRestartEtlShouldRespectRunFrequency()
                 Assert.True(result.Success);
                 Assert.False(result.Disabled);
 
-                etlDone = WaitForEtl(store, (n, statistics) => statistics.LoadSuccesses != 0);
+                Assert.True(WaitForDatabaseToUnlock(store, timeout: TimeSpan.FromMilliseconds(100), out var database));
```
Comment (ppekrol) [inline test/SlowTests/Server/Documents/ETL/Olap/LocalTests.cs]: isn't that to low? 100ms?

File: test/SlowTests/Server/Documents/ETL/Olap/LocalTests.cs [hunk 2]
```diff
@@ -970,6 +976,45 @@ public async Task AfterDatabaseRestartEtlShouldRespectRunFrequency()
...
+                    database = GetDocumentDatabaseInstanceFor(store).Result;
+                    return true;
+                }
+                catch (AggregateException e)
+                {
+                    if (e.Message.Contains($"The database '{store.Database}' has been unloaded and locked"))
+                        continue;
+                    
+                    throw;
+                }
```
Comment (ppekrol) [inline test/SlowTests/Server/Documents/ETL/Olap/LocalTests.cs:1015]: Do a small Thread.Sleep here? e.g. 13?

Comment (aviv86) [reply test/SlowTests/Server/Documents/ETL/Olap/LocalTests.cs:1015]: done

File: test/SlowTests/Server/Documents/ETL/Olap/LocalTests.cs (hunk 1 above)
Comment (aviv86) [reply test/SlowTests/Server/Documents/ETL/Olap/LocalTests.cs]: changed to 1000ms

**Pull request merged**